sources/packages/backend/apps/db-migrations/src/migrations/1717081412569-CreateECertErrorsReport.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class CreateECertErrorsReport1717081412569
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Create-eCert-errors-report.sql", "Reports"),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Drop-eCert-errors-report.sql", "Reports"),
```

Comment:
Please adjust the rollback file name as `Rollback-create-e-cert-errors-report.sql`